lib/clients/smapi-client/resources/alexa-hosted.js

```diff
@@ -18,7 +18,8 @@ module.exports = (smapiHandle) => {
             manifest: manifest.manifest,
             hosting: {
                 alexaHosted: {
-                    runtime
+                    runtime,
+                    region: manifest.region
```

Comment:
I think the function input parameter should not be named as `manifest`, but we need to change this function interface to `function createHostedSkill(hostedSkillPayload, callback)`, where the hostedSkillPayload should contain `{vendorId, manifest, runtime, region}`. Could you help fix that?